pytest_fixtures/xdist.py

```diff
@@ -0,0 +1,12 @@
+"""Fixtures specific to or relating to pytest's xdist plugin"""
+import pytest
+
+from robottelo.config import settings
+
+
+@pytest.fixture(scope="session", autouse=True)
+def align_xdist_satellites(worker_id):
+    """Set a different Satellite per worker when available in robottelo's config"""
+    settings.server.hostname = settings.server.get_hostname(worker_id)
```

Comment:
You need to check and set the configuration first before accessing the `settings` from `robottelo.properties` by :

```
if not settings.configured:
     settings.configure()
```
You are able to access those settings because other plugins above `pytest_fixtures.xdist` in `conftest.py` are setting it for you. But tomorrow they might swap the place or probably configurable whether to call or not. Hence its good to set it locally.

You can also choose to set it at the module level instead in the fixture function.

The plan is to fix it in future by setting it once for all the required modules.